components/closebutton/index.css

```diff
@@ -137,11 +117,12 @@ a.spectrum-CloseButton {
 	@extend %spectrum-AnchorButton;
 }
 
+/* stylelint-disable-next-line no-duplicate-selectors -- Allow variable definitions to appear at the top */
```

Comment:
Out of scope for this PR — but would it be worth disabling this listing rule since it's a pretty common issue?

Reply:
It's one we're eager to fix though, just not the time to do it in this PR. Post-S2 it won't be a problem anymore though!